indico_toolkit/metrics/usermetrics.py

```diff
@@ -0,0 +1,92 @@
+import pandas as pd
+from typing import List, Dict
+from indico import IndicoClient
+from datetime import datetime
+from indico_toolkit.indico_wrapper import IndicoWrapper
+from indico_toolkit import ToolkitInputError
+from indico.queries.usermetrics import GetUserSnapshots
+from indico.filters import UserMetricsFilter
+
+
+class UserMetrics(IndicoWrapper):
+    def __init__(self, client: IndicoClient):
+        self.client = client
+
+    def get_user_metrics_df(
+        self,
+        columns: List[str] = [
```

Comment:
could/should we make these arguments instance variables? It's just that I'm seeing columns, date, filter_user_id, and filter_email being used in both functions.

Reply:
I agree with Nate. This should be defined as a list at the top of the module.

Reply:
Removed option for columns in methods - static list on top of module now - doesn't save us any resources by decreasing any return data, so we're just returning everything now